components/footer_information/_footer_information.scss

```diff
@@ -0,0 +1,8 @@
+/*! @convivial_bootstrap/footer_information */
+
+.footer-information {
```

Comment:
You may need the padding even if the information area does not have the color palettes.
This could be changed or not changed depending on your preference.

Reply:
okay